packages/grid/_modules_/grid/hooks/utils/useThemeProps.ts

```diff
@@ -0,0 +1,20 @@
+import { getThemeProps } from '@material-ui/styles';
+import { useTheme } from '@material-ui/core/styles';
+
+// TODO: To be removed once we migrate to v5
```

Comment:
We could have hosted it in `packages/grid/_modules_/grid/utils/material-ui-utils.ts` with the other copied modules from v5, but no specific preferences

Reply:
Me neither. I saw that the helper "util" hooks were in that place that's why I added it there. It's better for them to be in the same place. Moved.

Reply:
Maybe if there are at the same place, it will be more likely for us to not forget any when the time come 😁